plugins/jreleaser/src/main/resources/org/jreleaser/cli/Messages_it.properties

```diff
@@ -193,9 +193,9 @@ jreleaser.release.tag-name                        = Tag del rilascio.
 jreleaser.release.previous-tag-name               = Tag della versione precedente.
 jreleaser.release.release-name                    = Nome del rilascio.
 jreleaser.release.milestone-name                  = Nome della milestone.
-jreleaser.release.prerelease                      = Se � un pre-rilascio.
+jreleaser.release.prerelease                      = Se � un pre-rilascio.
```

Comment:
Here it's just a tiny difference with the accent:

![Screenshot 2022-10-30 at 17 44 39](https://user-images.githubusercontent.com/105051647/198890662-9de499cf-20c4-439d-b771-083e562430e9.png)